source/uwp/Renderer/lib/AdaptiveImageRenderer.cpp

```diff
@@ -499,7 +452,14 @@ namespace AdaptiveCards::Rendering::Uwp
 
     // Issue #8127
     template<typename T>
-    void render_xaml::XamlBuilder::PopulateImageFromUrlAsync(winrt::Uri const& imageUrl, T const& imageControl, bool const& isImageSvg)
+    winrt::ImageSource render_xaml::XamlBuilder::PopulateImageFromUrlAsync(winrt::Uri const& imageUrl,
+                                                                           T const& uiElement,
+                                                                           bool isAutoSize,
+                                                                           winrt::IInspectable const& parentElement,
+                                                                           winrt::IInspectable const& imageContainer,
+                                                                           bool isVisible,
+                                                                           bool isImageSvg,
+                                                                           winrt::Stretch stretch)
```

Comment:
If this set of fields is passed around enough it might make sense to put them in a `struct` and pass that around instead.  If you are concerned about excess copies the struct could have the copy constructor/operator deleted.